wemulate/controllers/restore_controller.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Optional
+
+import typer
+
+import wemulate.controllers.common as common
+import wemulate.ext.utils as utils
+from wemulate.utils.output import err_console, console
+from wemulate.core.database.setup import pre_setup_database
+
+
+app = typer.Typer(help="restore connections and parameters stored in the database")
+
+
+@app.command(help="restore all connections", no_args_is_help=False)
+def device():
+    utils.restore_device()
+    console.print("device restored")
```

Comment:
I would change it to "_successfully restored device_" to follow the same scheme as in the connection function.